tests/test_main/test_pretrain.py

```diff
@@ -0,0 +1,154 @@
+from os.path import getmtime, join, exists
+
+from corebehrt.main.pretrain import main_train
+from corebehrt.common.setup import DATA_CFG, PRETRAIN_CFG
+
+from .base import TestMainScript
+
+
+class TestCreateOutcomes(TestMainScript):
+    def setUp(self):
+        super().setUp()
+
+        self.pretrain_dir = join(self.tmp_dir, "pretrain")
+
+        self.set_config(
```

Comment:
Shouldn't we be using the default test configs, instead of having a hardcoded one?

Reply:
The initial idea with hard-coding the config was to make it easy to change/re-create between tests. We also did not have test config files in the `tests` directory then.

I had to add them to the `tests/data/{features,tokenized,outcomes}` with the new `test_main/test_*`  tests, and if we are not changing them a lot, we can read them from there instead now :-)

Reply:
I think we should, be we can leave that for future as well